### ISSUE TITLE
Support <Choose>

### ISSUE THREAD

**Issue by texttechne** [vkbansal/eslint-plugin-jsx-control-statements#3]
In the next release of JSX-control-statements, we're introducing the choose statement to allow for else if. Example code:

```
<Choose>
  <When condition={this.props.when1}>
    <span>WhenBlock1</span>
  </When>
  <When condition={this.props.when2}>
    <span>WhenBlock2</span>
  </When>
  <Otherwise>
    <span>OtherwiseBlock</span>
  </Otherwise>
</Choose>
```
### Rules
- `<Choose>` may not be empty
- `<Choose>` must have one `<When>` at least
- `<Otherwise>` is optional, but if it is specified
  - it must come last 
  - and may only occur once
- `<When>` requires attribute `condition`

**Comment by vkbansal** [comment]
Let's decide the rule names:
- `jsx-choose-not-empty`. (not empty and at least one `<When>`)
- `jsx-otherwise-once-last`. (`<Otherwise>` must come once and be last)
- `jsx-when-require-condition`. (`<When>` requires attribute `condition`)

Edit: update `jsx-otherwise-last` to `jsx-otherwise-once-last`

**Comment by texttechne** [comment]
Only the second name doesn't seem to fit. 
`jsx-otherwise-once-last` maybe?

And just to be sure: `<Otherwise>` is optional, it does not have to occur.

**Comment by vkbansal** [comment]
That makes sense. I'll start working on it ASAP.

**Comment by texttechne** [comment]
Thank you very much!

**Comment by vkbansal** [comment]
See #5

**Comment by vkbansal** [comment]
Released in v1.1.0
